### DISCUSSION TITLE
Comparação de spreads

### DISCUSSION THREAD

**Discussion by igorlvicente** [DyegoCosta/trabalhando-remoto#137] [Show and tell]
⚠️ **Edit: Não estou mais atualizando esse post. Atualmente uso o Husky.io e não fui mais atrás de comparar spread, já que o Husky me serve bem e cobra pouco. Quem quiser continuar alterando a discussão, fique à vontade.**

---

Podemos ter uma comparação de spreads reais para avaliarmos os serviços de conversão? Porque os serviços podem anunciar o valor que usam de spread, mas dependendo da cotação usada por eles, o spread pode ser bem maior. Dependendo do valor convertido, esse spread também pode variar.

Então eu gostaria que vocês postassem as seguintes informações das conversões que vocês fizeram ao longo do tempo:
- **Cotação final**: O valor em reais que entrou na sua conta dividido pelo valor em dólares que o empregador te enviou
- **Serviço utilizado**: Não se esqueça de dizer se é PF ou PJ
- **Data da conversão**

Repare que essas informações não mostrarão o quanto vocês ganharam, então não tem problema compartilhar.

---
Aqui está um template que pode ser usado:
```markdown
|Cotação Final|Serviço|Data|
|---|---|---|
|XX,00|Payoneer Online|2021-01-1|
|XX,00|Payoneer Online|2021-01-1|
```
Os meus valores são:

|Cotação Final|Serviço|Data|
|---|---|---|
|5,5727|Remessa Online PJ|2021-03-16|
|5,6981|Remessa Online PJ|2021-04-12|
|5,3641|Remessa Online PJ|2021-05-05|

---

A imagem abaixo se aproxima do que eu quero fazer. Nela, há 3 conversões que eu fiz e 1 conversão que um amigo meu fez (por coincidência, no mesmo dia que uma das minhas).

![image](https://user-images.githubusercontent.com/61885036/118101843-6da2cf80-b3ae-11eb-88be-589c9c24fe9c.png)

---

Lista:

|Usuário|Cotação Final|Serviço|Data|
|---|--:|---|---|
||||**Março de 2021**|
|@pierresantana|5.6100 | Branco do Brasil (PJ) | 2021-03-03|
|@igorlvicente|5,5727|Remessa Online PJ|2021-03-16|
||||**Abril de 2021**|
|@pierresantana|5.5100 | Branco do Brasil (PJ) | 2021-04-08|
|@igorlvicente|5,6981|Remessa Online PJ|2021-04-12|
||||**Maio de 2021**|
|@pierresantana|5.3812 | Branco do Brasil (PJ) | 2021-05-03|
|@igorlvicente|5,3641|Remessa Online PJ|2021-05-05|
|@lucas-barros|5,2450|BS2 PJ|2021-05-25|
||||**Junho de 2021**|
|@igorlvicente|4,9371 | Remessa Online | 2021-06-08|
|@pierresantana|5.0067 | Branco do Brasil (PJ) | 2021-06-08|
|@lucas-barros|5,0190|BS2 PJ|2021-06-10|

**Comment by vdurante** [comment]
Recebi aqui do BS2 que para euro eles cobram spread de até 2% + 20 dólares por recebimento

**Comment by lucas-barros** [comment]
|Cotação Final|Serviço|Data|
|---|---|---|
|5.245|BS2 PJ|2021-05-25|

**Comment by igorlvicente** [comment]
|Cotação Final|Serviço|Data|
|---|---|---|
|4,9371|Remessa Online|2021-06-08|

**Comment by pierresantana** [comment]
Cotação Final | Serviço | Data
-- | -- | --
5.6100 | Branco do Brasil (PJ) | 2021-03-03
5.5100 | Branco do Brasil (PJ) | 2021-04-08
5.3812 | Branco do Brasil (PJ) | 2021-05-03
5.0067 | Branco do Brasil (PJ) | 2021-06-08

**Reply by igorlvicente** [reply]
Obrigado e continue postando.

**Reply by pierresantana** [reply]
Uma informação importante. O spread que consegui, comparando o valor no site https://www.investing.com/currencies/usd-brl, deu menos de 1 centavo de diferença, mas o BB cobra uma taxa por operação de cambio. O valor é R$ 550,00, mas eu consegui com o meu gerente de conta reduzir pela metade. Mesmo com essa taxa ainda não vi vantagem de sair do BB e usar outros serviços de cambio.

**Reply by igorlvicente** [reply]
Mas só pra entender, as cotações postadas por você levam em conta essa taxa?

**Reply by pierresantana** [reply]
Sim, ali é o VET (valor efetivo total):

Cotação final = ((spread * valor recebido em dolar) - taxa) / valor recebido em dolar

**Comment by lucas-barros** [comment]
Cotação Final | Serviço | Data
-- | -- | --
5,0190 | BS2 PJ | 2021-06-10

Hoje nao consegui falar com a pessoa que normalmente falo, entao liguei para o numero generico de cambio deles (11 2103 7924).
Inicialmente me deram um cambio de 4.93. Eu achei estranho e falei que normalmente o valor do spread era menor, ai depois ele me passou essa valor de 5.019. Resumindo, parece ter um valor "oficial" e um valor com "desconto".

**Reply by igorlvicente** [reply]
Bom saber.

**Comment by bolds07** [comment]
Essa comparação aqui está completamente escrota, tem validade nenhuma porque cada um chega e bota uma cotação num dia diferente de um site diferente sem referencia com a cotaçao do banco central NA MESMA HORA. simplesmente serve de nada

O VALOR DO DOLAR ALTERA A CADA SEGUNDO

colocar um dia la pode ter uma margem de erro de 4%

**Reply by igorlvicente** [reply]
A ideia é comparar com o maior e o menor valor do banco central no dia. Não vai dar uma informação precisa, como você disse, mas é melhor que nada.

**Comment by bolds07** [comment]
FIZ A COTAÇÃO COM 3 BANCO QUE EU TENHO CONTA E TENHO SALDO A RECEBER

o valor que vou colocar aqui é o VALOR LIQUIDO, apos iof e todos os custos
cotação feita no dia 22/06 as 11h40

4,9614 inter
4,8988 bs2
4,8593 remessa online

O remessa online é disparado o pior de todos que eu comparei, é inacreditavel como tem tanta gente nesse forum babando o ovo deles...

a simulação foi feita com VALORES REAIS tenho saldo em cada um desses bancos e fiz o resgate de 5mil dolares tudo referente a adsense

**Reply by igorlvicente** [reply]
Fui conferir o Inter e só dá pra receber acima de 5000 dólares de uma vez. Sem contar que tive que entrar em contato com eles pra saber isso, pois não vi nenhuma opção no aplicativo.
(Isso tudo como PJ).

Checarei o BS2.

**Reply by igorlvicente** [reply]
E estou simulando agora mesmo:
- Remessa Online PJ: 4,8166
	- Com o cupom de desconto que eles dão pode ficar maior
- BS2 Conversão na Conta Internacional PJ: 4,7964
	- Com a taxa de 12 USD para receber o dinheiro, esse valor pode ficar menor

Nesse caso o Remessa online tem mais vantagem. Estou fazendo algo errado?

**Comment by ericcoutinho** [comment]
Que cupom é esse do remessa? Tenho que solicitar?

Em qua., 30 de jun. de 2021 às 05:04, Igor L. Vicente <
***@***.***> escreveu:

**Reply by igorlvicente** [reply]
@ericcoutinho, toda vez que eu converto por lá, no próximo mês eu recebo. É um cupom de "fidelidade". Até achei bem pensado da parte deles (dar o cupom só depois da primeira conversão pra manter você convertendo).

Eles enviam via e-mail.

**Comment by lucas-barros** [comment]
Cotação Final | Serviço | Data
-- | -- | --
4.94 | BS2 PJ | 2021-06-30

**Comment by vdurante** [comment]
Eu mudaria a estrutura para:

| Cotação Comercial | Cotação Final | Spread | Serviço |Data |
|---|---|---|---|---|
|7,00|6,72|4%|Banco Exemplo|2021-01-1|

E o spread seria calculado: `1-(Final/Comercial)`

O valor da moeda comercial teria que ser consultada no google ou similar. É muito importante esta parte pois alguns serviços cotam uma moeda comercial diferente da internacional.

**Reply by igorlvicente** [reply]
Mas é bem isso aí mesmo, @vdurante:
`Cotação final: O valor em reais que entrou na sua conta dividido pelo valor em dólares que o empregador te enviou`

**Reply by vdurante** [reply]
vdd, fui desatento! Então acho que ta top! Se tiver a cotação comercial do momento fica perfeito

**Reply by igorlvicente** [reply]
A minha ideia é depois fazer uma planilha que busca a cotação no momento, mas até agora não dei a prioridade necessária pra fazer.

**Comment by vdurante** [comment]
Vou manter este comentário atualizado com minhas cotações

| Serviço | Data | Cotação Comercial | Cotação Final | Spread |
|---|---|---|---|---|
| BS2 | 12/07/2021 | 6,20 | 6,027 | 2,8% |

**Reply by borgateo** [reply]
2,8% acho caro 🤔

**Comment by 123dma** [comment]
Utilizo o banco Rendimento
|Cotação Final|Serviço|Data|Cotação Comercial
|---|---|---|---|
|5,1249|Banco Rendimento|22/07/2021|5,1990|

**Comment by valterlima** [comment]
<html>
<body>


Serviço | Data | Cotação Comercial | Cotação Final | Spread
-- | -- | -- | -- | --
Husky | 06/08/2021 | 5.2185 | 5.166 | 1.1%
Husky | 03/09/2021 | 5.1608 | 5.109 | 1.0%


</body>
</html>

**Comment by cesdias** [comment]
| Serviço | Data | Cotação Comercial | Cotação Final | Spread |
| --- | --- | --- | --- | --- |
| Husky | 09/09/2021 | 5.3082 | 5.2551 | 1.0% |

**Comment by pablo-sg-pacheco** [comment]
| Serviço        | Moeda | Data       | Cotação comercial | Cotação final | Spread |
|----------------|-------|------------|-------------------|---------------|--------|
| Remessa online | EUR   | 06/12/2021 | 6,39              | 6,26          | 1,981% |

**Comment by rafagan** [comment]
Eu uso o transferbank! Eles tem um spread fixo de 0,70% (o menor que já encontrei)!! E não tem isso de ficar ligando pra pedir cotação, eu posso fazer o câmbio a qualquer momento sozinho pela plataforma deles, e ainda tem uma função de taxa alvo, onde eu posso cadastrar uma tx que quero atingir e fecha automaticamente quando chegar nessa tx, por exemplo o dolar ta em 5,10 e eu gostaria de converter quando chegasse em 5,20, assim que bater 5,20 a conversão é feita automaticamente e eu n preciso me preocupar com nada! 
Além do melhor spread, sou muito bem atendido e uso uma plataforma digna (n rola mais fechar câmbio pelo telefone ne)..

https://www.transferbank.com.br/prestacao-de-servico-ao-exterior

**Reply by megurost** [reply]
Parece ser um serviço interessante, não conhecia. Mas só um toque, nesses casos seria legal comentar que você trabalha pra essa empresa que recomenda. Não que isso desmereça o serviço, claro.

**Reply by lucasmpaim** [reply]
O transferbank realmente tem um custo mais atrativo e a plataforma funciona muito bem. Eu particularmente uso muito a opção de taxa alvo.

![image](https://user-images.githubusercontent.com/7849484/168463869-81ba8b72-218f-4883-8c97-e3431d57cdfc.png)

**Comment by brnpimentel** [comment]
Sempre usei a husky e testei o **transferbank** agora no último pagamento. 

Gostei do serviço! Desde o atendimento inicial com o Guilherme, quanto o pós recebimento. Ele tirou todas minhas dúvidas via Whatsapp. O spread foi de 0,7%, e o dinheiro caiu em D+1. Foi bem transparente.

Gostei da idéia fazer o fechamento manual, quando o dinheiro chega você mesmo escolhe a cotação (em outras plataformas a cotação é "fechada" pela própria plataforma, podendo ter um gap ai, pegando a menor cotação do dia!)

Para receber na transferbank tem que anexar uma invoice que pode ser gerado em qualquer gerador por ai. E o spread de 0,7% é o menor que eu encontrei. Já testei husky e remessa online inclusive. 

Quem quiser testar e usar pode usar um código que o Guilherme gerou: **BMPA3** . A 1º conversão tem isenção de tarifa ($10) e spread de **0,5%**! 

[Onboarding Transferbank](https://onboarding.transferbank.com.br/codepartner=BMPA3)

Ah o whatsapp do Guilherme é: https://wa.me/554192855088 

**ps: não trabalho na transferbank! feedback meu mesmo!**
